schemas/multimodal/build.py

```
@@ -0,0 +1,228 @@
+    if not protoc_version_parts:
+        raise RuntimeError(
+            "Unable to parse protoc version from: %s" % protoc_version
+        )
+
+    if protoc_version_parts <= MINIMUM_PROTOC_VERSION:
+        raise RuntimeError(
+            "Unsupported protoc version %s at %s. Expected libprotoc > 33."
+            % (
+                protoc_version,
+                protoc_path,
+            )
```

Ruff (0.15.10)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)